packages/node/src/protocol/install-virtual-app.ts

```diff
@@ -232,7 +232,7 @@ function createAndAddTarget(
     0, // app nonce
     defaultTimeout,
     {
-      playerAddrs: [initiatingAddress, intermediaryAddress],
+      playerAddrs: [initiatingAddress, respondingAddress],
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll can you ack this

Reply:
LGTM